support/tests/test_tealium_debug/TealiumDebugServerTests.swift

```diff
@@ -49,6 +50,28 @@ class TealiumDebugServerTests: XCTestCase {
         
     }
     
+    func testQueueMaxLimit () {
+      
+        let debugServer = TealiumDebugServer()
+        debugServer.setQueueMax(50)
+        
+        for i in 1..<53 {
+            debugServer.add(["\(i)": "\(i)"])
+        }
+      
+        XCTAssertTrue(debugServer.debugQueue.count == debugServer.queueMax, " expected count of  \(debugServer.queueMax) does not match queue count:  \(debugServer.debugQueue.count)")
+        
+        //first item should be removed
+        if (debugServer.debugQueue.contains { $0 == ["1":"1"] } == true) {
+            XCTFail("queue is not compliant with FIFO \(debugServer.debugQueue.description)")
+        }
+        
+        //last item should be the last item added
+        if (debugServer.debugQueue.contains { $0 == ["52":"52"] } == false) {
```

Comment:
This checks for a value that should be in the array, but doesn't check for expected position.